<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 44, 54);
color:white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>FreeSerif</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(39, 44, 54);
color:white;</string>
     </property>
     <property name="title">
      <string>PARAMETRES DE VIDEO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>332</width>
        <height>104</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>330</width>
         <height>102</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout">
          <property name="labelAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white ;</string>
            </property>
            <property name="text">
             <string>FRAME  1 :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="styleSheet">
             <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: white;
color:black ;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white ;</string>
            </property>
            <property name="text">
             <string>FRAME  2 :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="styleSheet">
             <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: white;
color:black ;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>351</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="frame1_layout"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>341</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="frame2_layout"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>170</y>
        <width>351</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="diff_layout"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>214</width>
        <height>38</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">.QPushButton
{
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white 
}
QPushButton:pressed{ background-color: blue; }</string>
         </property>
         <property name="text">
          <string>FERMER</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">.QPushButton
{
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white 
}
QPushButton:pressed{ background-color: blue; }</string>
         </property>
         <property name="text">
          <string>DIFFERENCE</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
